### ISSUE TITLE
Insufficient USB Permissions

### ISSUE THREAD

**Issue by Crsarmv7l** [atlas0fd00m/rfcat#135]
Hi,

I recently got a YS1, set everything up, updated the FW on the YS1, added the udev rules, and $USER is a member of dialout, but I still get 

```
Error in resetup():USBError(13, 'Access denied (insufficient permissions)')
```
I have done this on two different computers, with two different distros (debian and arch)

YS1 works fine with sudo, but the error is bugging me

**Comment by sdb333777** [comment]
you tried using root account?

**Comment by Crsarmv7l** [comment]
> you tried using root account?

Did you read what I wrote?

>YS1 works fine with sudo, but the error is bugging me

**Comment by atlas0fd00m** [comment]
that's an odd one.  the udev rules should be sufficient, if you've either rebooted or run `udevadn control -R` as root since installing the udev rules (correctly).

any other interesting things?  what versions of arch and debian?

**Comment by Crsarmv7l** [comment]
Normal stable debian, and Manjaro. There is a commonality; I am running Gnome DE on both.

**Comment by Crsarmv7l** [comment]
> that's an odd one. the udev rules should be sufficient, if you've either rebooted or run `udevadn control -R` as root since installing the udev rules (correctly).
> 
> any other interesting things? what versions of arch and debian?

Don't know what is going on with debian right now, but I have arch working. I decided to actually look at the udev rules more closely and lets just say I am an idiot 🤣 , of course it wouldn't work! It was looking for GROUP="dialout". Arch doesn't have dialout, it has uucp.

Changed it, reloaded the udev rules, and I did have some odd behavior but it is working without sudo
